apps/web/public/locales/es.json

```diff
@@ -1101,7 +1101,7 @@
   "this role/badge?": "este rol/insignia?",
   "title: text": "título : texto",
   "to set it up.": "para configurarlo.",
-  "topics: up to 3 topic names separated by spaces and/or commas e.g. “food organic”": "temas: hasta 3 nombres de temas separados por espacios y/o comas, p. \n“comida organica”",
+  "topics: up to 3 topic names separated by spaces and/or commas e.g. “food organic”": "temas: hasta 3 nombres de temas separados por espacios y/o comas, p. \n"comida organica"",
```

Comment:
this looks broken

Reply:
yeah, cursor was doing this weird thing when I was asking it to translate completely unrelated lines; fixed it again